pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -77,6 +87,18 @@ func (c *connectivityCheckController) Sync(ctx context.Context, syncContext fact
 		return nil
 	}
 
+	enabled, err := Enabled(operatorSpec)
+	if err != nil {
+		return err
+	}
+
+	if !enabled {
+		// TODO delete CRD
```

Comment:
I can't let you leave this as  TODO.  We need this resolved for our 4.6 release.

Reply:
Done.